nwbwidgets/placefield.py

```diff
@@ -0,0 +1,501 @@
+import numpy as np
+from scipy.ndimage import label
+from scipy.ndimage.filters import gaussian_filter, maximum_filter
+
+import matplotlib.pyplot as plt
+import nelpy.plotting as nlp
```

Comment:
putting this import here would require us to list `nelpy` as a dependency of this project. I try not to include dependencies unless we have to. What methods are we using form nelpy? Do we need them? If so, see SpikeInterface's way of dealing with these types of dependencies: https://github.com/SpikeInterface/spikeextractors/blob/5b9851c3ef8abd3720b2f97d30bc26bc0511f8b8/spikeextractors/extractors/neuroscopeextractors/neuroscopeextractors.py#L9-L14

Reply:
In this case it looks like it's just doing simple matplotlib tick manipulation, so I don't think it's worth adding nelpy as a dependency